src/main/cc/maze/cell/cell.c

```diff
@@ -0,0 +1,23 @@
+#include "cell.h"
+
+Cell Cell::Cell(int x_loc, int y_loc) {
+	x = x_loc;
+	y = y_loc;
+	visited = false; 
+}
+
+int Cell::GetLocationX(void) {
+	return x; 
+}
+
+int Cell::GetLocationY(void) {
+	return y; 
+}
+
+bool Cell::IsVisited(void) {
+	return visited; 
+}
+
+void Cell::VisitCell(bool state) {
```

Comment:
I feel like visit cell should take no parameters. Then in the cell we would do whatever we need and then set it to visited. Then we can also have a unvisit method